extensions/access_log_policy/config/v1alpha1/access_log_policy_config.proto

```diff
@@ -15,10 +15,19 @@
 
 syntax = "proto3";
 
+// $title: AccessLogPolicy Config
+// $description: Configuration for AccessLogPolicy Filter.
+// $location:
+// https://istio.io/docs/reference/config/proxy_extensions/accesslogpolicy.html
+// $weight: 20
+// This filter logs all error requests. Success requests are repeatedly logged
```

Comment:
This filter is used in conjunction with another filter.
Can you add more details here?
Which attribute is produced, and how another filter is expected to use it.

Reply:
Done.

Reply:
Accesslog Policy plugin is a stateful http log sampler. It decides whether a request is logged based on the following rules.
1. All requests resulting in errors are logged.
2. First successful request within `log_window_duration ` from a specific source ip (source principal) is  logged.

The plugin records its decision in the `access_log_policy` with a value of  `"no"`. A downstream plugin may honor the  the attribute. For example, `Stackdriver` plugin will not produce an access log entry if this attribute is set.

Reply:
Thanks! Changed.. this is much better :)